spec/models/organisation_spec.rb

```diff
@@ -68,4 +68,34 @@
       ])
     end
   end
+
+  context 'when organisation service email is left blank' do
+    let(:organisation) { build(:organisation, service_email: '') }
+
+    it 'is not valid' do
+      expect(organisation).not_to be_valid
+    end
+
+    it 'explains why it is invalid' do
+      organisation.valid?
+      expect(organisation.errors.full_messages).to eq([
+        "Service email can't be blank"
```

Comment:
I see we protect from nil but empty string is ok.
When does this happen as a user is using the site?

Reply:
@emileswarts Not sure if I've understood your comment/question correctly, but a user can also update their service email after confirming their account.